facefusion/face_masker.py

```diff
@@ -7,20 +7,48 @@
 import onnxruntime
 
 import facefusion.globals
-from facefusion.typing import FaceMaskType, Frame, Mask, ModelValue, Padding
+from facefusion.typing import FaceMaskType, Frame, Mask, ModelValue, Padding, FaceMaskParserRegion
 from facefusion.filesystem import resolve_relative_path
 from facefusion.download import conditional_download
 
 FACE_OCCLUDER = None
+FACE_PARSER = None
 THREAD_LOCK : threading.Lock = threading.Lock()
 MODELS : Dict[str, ModelValue] =\
 {
 	'occluder':
 	{
 		'url': 'https://github.com/facefusion/facefusion-assets/releases/download/models/occluder.onnx',
 		'path': resolve_relative_path('../.assets/models/occluder.onnx')
+	},
+	'parser':
+	{
+		'url': 'https://filebin.net/c090an8lf7ppgh91/parser.onnx',
```

Comment:
Model is online: https://github.com/facefusion/facefusion-assets/releases/download/models/face_parser.onnx